pkgs/applications/misc/gsctl/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  pname = "gsctl";
+  name = "${pname}-${version}";
```

Comment:
```suggestion
```

automatically set when those attributes `pname` and `version` are set.

Reply:
Done